src/components/table-header.js

```diff
@@ -52,6 +62,21 @@ const TableHeader = ({
     const currentTime = dateFormat(timestamp, 'HH:MM');
     const outlined = taskStatus === 'RUNNING' ? 'outlined' : 'filled';
     const tableHeaderName = (processName || '') + ' Supervisor';
+    const [manualExportEnabled, setManualExportEnabled] = useState([]);
+
+    useEffect(() => {
+        async function getManualExportEnabled() {
+            let data = {};
+            try {
+                const response = await fetch('process-metadata.json');
+                data = await response.json();
+            } catch (error) {
+                console.error('An error has occurred:', error);
+            }
+            setManualExportEnabled(data.manualExportEnabled || false);
```

Comment:
moove this line after 
data = await response.json();

and delete the line with
let data = {};